modules/ROOT/pages/6.0-release-notes-apis.adoc

```diff
@@ -0,0 +1,209 @@
+= TinyMCE 6.0 API changes
+:navtitle: TinyMCE 6.0 API changes
+:description: TinyMCE 6.0 API changes
+:keywords: releasenotes, apis
+
+This section documents the improvements, changes and fixes made to {productname} 6.0 APIs.
+
+It also documents the {productname} 6.0 APIs that have been removed.
+
+
+* xref:apis[APIs]
+* xref:new-and-improved-apis[New and improved APIs]
+* xref:changed-apis[Changed APIs]
+* xref:fixed-apis[Fixed APIs]
+* xref:removed-or-deprecated-apis[Removed or deprecated APIs]
+
+// tag::apis[]
+[[apis]]
+== APIs
+
+[[new-and-improved-apis]]
+=== New and improved APIs
+
+[[editor-annotator-removeall]]
+==== `editor.annotator.removeAll`
+
+The new `editor.annotator.removeAll` API replaces the `removeAll` function built-in to the Comments plugin.
+
+[[editor-options]]
+==== `editor.options` and `tinymce.activeEditor.options`
+
+The new `editor.options` API replaces two old APIs: `editor.settings` and `editor.getParam`.
+
+Along with this new API, the new `tinymce.activeEditor.options` registry replaces `tinymce.activeeditor.settings`.
+
+[[editor-uploadImages]]
+==== `editor.uploadImages()`
+
+The `editor.uploadImages()` API now includes a `remove` flag.
+
+The flag returns `true` when an image being uploaded is removed after failing to upload.
+
+[[fakeclipboard]]
+==== `FakeClipboard`
+
+The new `FakeClipboard` API provides a means for storing state that can be set and accessed by plugins and the core editor.
+
+`FakeClipboard` is implemented on the {productname} global, allowing anything to write to it and read from it.
+
+[[resource-unload]]
+==== `Resource.unload`
+
+The new `Resource.unload` API makes it possible to unload resources in the resource loader.
+
+This allows tests, such as `ImageEmojiTest`, to remove existing databases before loading their own.
```

Comment:
I don't think this is a good example for the end-users. Can we come up with something that is more related to the end users.

Reply:
Yeah `ImageEmojiTest` is very much an internal thing so we don't want to say something like that.

Reply:
```suggestion
```
Deferring figuring out a good example here